src/conversions/rgb_cmyk_conversion.rs

```diff
@@ -0,0 +1,56 @@
+/// Author : https://github.com/ali77gh\
+/// References:\
+/// RGB:  https://en.wikipedia.org/wiki/RGB_color_model\
+/// CMYK: https://en.wikipedia.org/wiki/CMYK_color_model\
+
+/// This function Converts RGB to CMYK format
+///
+/// ### Params
+/// * `r` - red
+/// * `g` - green
+/// * `b` - blue
+///
+/// ### Returns
+/// (C, M, Y, K)
+pub fn rgb_to_cmyk(r: u8, g: u8, b: u8) -> (u8, u8, u8, u8) {
```

Comment:
What do you think about
```suggestion
pub fn rgb_to_cmyk(rgb: (u8, u8, u8)) -> (u8, u8, u8, u8) {
```

Reply:
Yeah. It's much better. 
It's easier to pass around a tuple instead of repeating r g b everywhere!

Cool👍